cql-ast/src/main/scala/troy/cql/ast/dml/Condition.scala

```diff
@@ -3,3 +3,7 @@ package troy.cql.ast.dml
 import troy.cql.ast.Term
 
 final case class Condition(simpleSelection: SimpleSelection, operator: Operator, term: Term)
+
+sealed trait IfCondition
+final case class SimpleIfCondition(conditions: Seq[Condition]) extends IfCondition
+final case class Exist(value: Boolean) extends IfCondition
```

Comment:
I refactored these classes in [another commit ](https://github.com/cassandra-scala/troy/pull/62/commits/608a78296abdd3ad4eac57028518203caf8a1b11)